apollo-backend/src/test/java/io/logz/apollo/helpers/StandaloneApollo.java

```diff
@@ -85,6 +85,11 @@ public void startKubernetesMonitor() {
         kubernetesMonitor.start();
     }
 
+    public void stopKubernetesMonitor() {
```

Comment:
Not sure how it is related, can't really comment on that

Reply:
Started to write tests, and figured i can't actually :)
Just leaving this as someone will need this in the future